packages/angular/package.json

```diff
@@ -6,28 +6,28 @@
     "ng": "ng",
     "dev": "ng build --watch",
     "start": "ng serve",
-    "build": "ng build --prod",
+    "build": "ng build --configuration production",
```

Comment:
Doesn't Angular default to production build? Or is this still needed with 12?

Reply:
Technically, `ng build` will default to dev build unless specified, according to 

https://github.com/aws-amplify/amplify-ui/blob/14ad7bf8125491ebfa91800f2b5611e158ac8ce2/packages/angular/angular.json#L14-L23

Reply:
this might be library-only thing, but that's our default.